script/run-unit-test.js

```diff
@@ -33,23 +36,23 @@ if (
   coverageInclude = `--include 'src/applications/${options['app-folder']}/**'`;
 }
 
+const reporterOption = options.reporter ? `--reporter ${options.reporter}` : '';
+
 if (options.help) {
   printUnitTestHelp();
   process.exit(0);
 }
 
-const mochaPath = 'BABEL_ENV=test mocha';
+const mochaPath = `BABEL_ENV=test mocha ${reporterOption}`;
 const coveragePath = `NODE_ENV=test nyc --all ${coverageInclude} --reporter=lcov --reporter=text --reporter=json-summary mocha --reporter mocha-junit-reporter --no-color`;
 const testRunner = options.coverage ? coveragePath : mochaPath;
 const mochaOpts =
   'src/platform/testing/unit/mocha.opts src/platform/testing/unit/helper.js';
 
-// Otherwise, run the command
 runCommand(
   `LOG_LEVEL=${options[
     'log-level'
   ].toLowerCase()} ${testRunner} --max-old-space-size=4096 --opts ${mochaOpts} --recursive ${options.path
     .map(p => `'${p}'`)
     .join(' ')}`,
-  options.coverage ? null : 0,
```

Comment:
@jbalboni: Since you wrote this originally, I want to make sure this isn't going to break some expected behavior. I wasn't sure why the error status should be suppressed when just running `mocha` without the coverage report.

Reply:
I think I did that because initially we were just running `--coverage` locally and not in CI, and it was not helpful to show the failure output when running coverage reports locally. I wouldn't expect this to break anything